docs/xdump.md

```diff
@@ -564,7 +564,8 @@ You can filter the **slow** event to change the time threshold from the default
 ```
 
 #### allocation event
-**(Linux, macOS, and Windows only)**
+
+**(Linux, macOS, Windows, and z/OS only)**
```

Comment:
It should be all platforms, it's not platform specific.

Reply:
Well, all platforms that use OpenJ9

Reply:
OK. Removed restriction completely. Should be ready to merge